examples/pdc_test_cache.rb

```diff
@@ -47,3 +47,6 @@ def main
 end
 
 main if __FILE__ == $PROGRAM_NAME
+
+a = 'aaaaa'
+flie = "bbbb"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)